pkgs/by-name/funkwhale/package.nix

```
@@ -0,0 +1,231 @@
+{
+  lib,
+  python3,
+  fetchFromGitLab,
+  postgresql,
+  postgresqlTestHook,
+  redisTestHook,
+  funkwhale,
+  runCommand,
+
+  # Frontend
+  stdenv,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  nodejs,
+
+  nix-update-script,
+}:
+let
+  python = python3;
+
+  meta = {
+    description = "Federated platform for audio streaming, exploration, and publishing";
+    homepage = "https://www.funkwhale.audio/";
+    changelog = "https://docs.funkwhale.audio/changelog.html";
+    license = lib.licenses.agpl3Only;
+    teams = [ lib.teams.ngi ];
+    mainProgram = "funkwhale-manage";
+  };
+in
+python3.pkgs.buildPythonApplication rec {
```

Nixpkgs manual suggests to use `python3Packages` (https://nixos.org/manual/nixpkgs/stable/#buildpythonapplication-function) but I am not sure if this makes any difference. 

```suggestion
python3Packages.buildPythonApplication rec {
```